Cargo.toml

```diff
@@ -5,12 +5,14 @@ edition = "2021"
 
 [lib]
 name = "frequenz_resampling"
-crate-type = ["rlib"]
+crate-type = ["cdylib", "rlib"]
 
 [features]
+#python = ["pyo3/extension-module"]
```

Comment:
Maybe this can be uncommented in the first commit?

Reply:
This is already uncommented.

Reply:
Ah I see, I will recreate the commits and tidy things up.

Reply:
Changed it!